source/Octopus.Tentacle.Tests.Integration/Support/TentacleFetchers/NugetTentacleFetcher.cs

```diff
@@ -71,18 +71,23 @@ async Task<string> DownloadAndExtractFromUrl(string directoryPath, TentacleRunti
             return Path.Combine(tentacleFolder, "tentacle", "Tentacle");
         }
 
-        public string[] TentacleArtifactNames(TentacleRuntime runtime)
+        public string[] TentacleArtifactNames(Version version, TentacleRuntime runtime)
         {
             if (PlatformDetection.IsRunningOnWindows)
             {
+                // Tentacle 8.2 is/was built on .NET8, previous versions were built on .NET6
+                string runtimeForVersion = version.Major >= 8 && version.Minor >= 2
+                    ? RuntimeDetection.DotNet8
+                    : RuntimeDetection.DotNet6;
```

Comment:
With everything on .NET 8 we might not need this any more.

Reply:
This one is sus. 2 attempts to remove the NET6 reference resulted in a massive test failure.
-https://github.com/OctopusDeploy/OctopusTentacle/pull/991/commits/42fd80ad0b46dd83310b85007ce87d342bfc864a#diff-902a65266fdc53157ba08edabaff0a144b82f643004614fd6aabd3cb72f247e7
- https://github.com/OctopusDeploy/OctopusTentacle/pull/991/commits/a0c7f459d563ef54145cca5a7601dd8121cd91c4#diff-902a65266fdc53157ba08edabaff0a144b82f643004614fd6aabd3cb72f247e7

CC: @acodrington

Reply:
At this point, I'll revert the change first to get a green build.

Reply:
This is likely to be the reason, https://docs.google.com/document/d/1bVnkDoctYpf-GdkoAPzGTc5Oyj_O2NlwqRl43Gv3MwM/edit#bookmark=id.976uxf7b549q
- I didn't do any further investigation though.